main.go

```diff
@@ -351,6 +356,8 @@ func main() {
 				opts = append(opts, grpc.ChainStreamInterceptor(streamInterceptors...))
 				opts = append(opts, grpc.ChainUnaryInterceptor(unaryInterceptors...))
 
+				grpc_prometheus.EnableHandlingTimeHistogram(grpc_prometheus.WithHistogramBuckets(durationBuckets))
```

Comment:
The go-grpc-prometheus docs mention that this can be expensive for prometheus servers. Is this likely to be problematic? Also, does this add significant load to bazel-remote?

Reply:
It mentions it as a precaution but I don't believe this will cause significant load. Do you prefer to put the handler metrics behind a option flag ? `--enable-endpoint-metrics`

Reply:
A flag would be nice for people that don't use the metrics.

Reply:
done !! :-)